libexec/ramalama/ramalama-serve-core

```diff
@@ -1,9 +1,15 @@
 #!/usr/bin/env python3
 
+import argparse
 import sys
 
 
 def main(args):
```

Comment:
This will come into play more in future PRs, it will become the proxy server for llama.cpp, vllm, etc.